src/System.Private.ServiceModel/tests/Scenarios/Binding/Custom/Interceptingbindingelement.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
+using System.ServiceModel.Channels;
+
+public class InterceptingBindingElement : BindingElement
```

Comment:
The name of this file is not correctly capitalized (InterceptingbindingElement.cs)